### PR TITLE
switch to docker based services

### PR DESCRIPTION
**Pull request by jmglsn**
- simplify travis builds
- same environment on all machines, less dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,12 +66,9 @@ Currently it's running on `ActiveMq` (Port 61010), `Apollo` (61020), `RabbitMq`
...
 While ActiveMq must be configured to use no login at all.
 
-You can setup all brokers by running `travisci/bin/ci/setup.sh`. Stop them by `travisci/bin/ci/stop.sh`.
-Used version can be configured at `travis/(BROKER)_VERSION`.
+You can setup all brokers by running `travisci/bin/start.sh`. Stop them by `travisci/bin/stop.sh`. (Docker is required.)
 
-You can use `travisci/bin/ci/clean.sh` to reset the test environment.
-
-If you only like to run the functional generic tests, ensure Apollo is configured. 
+If you only like to run the functional generic tests, ensure Apollo is configured.
```
Comment (staabm) [inline README.md]: Should note how/where to check that

**Comment by jmglsn** [comment]
Hopefully we locate the bug now, will continue tomorrow... please don't merge until we know at what point it breaks. I added some debug stuff (...)
Regarding docker, that parts that we use don't contain that much magic :)

**Comment by jmglsn** [comment]
https://gist.githubusercontent.com/jmglsn/2957acbddffff6a94c0cb8eb3c9a56bd/raw/f7747bd45964050194e73a357639495bee9dece5/no-output-001.log

**Comment by jmglsn** [comment]
This should be superfluous now, as it's part of #79 now.

**Pull request closed without merging**